plonky3/src/verifier.rs

```diff
@@ -173,31 +195,12 @@ where
     let quotient_domains: Vec<Vec<_>> = tables
         .values()
         .zip_eq(proof.opened_values.values())
-        .map(|(input, opened_values)| {
-            let degree = 1 << opened_values.log_degree;
-            let trace_domain = pcs.natural_domain_for_degree(degree);
+        .zip_eq(trace_domains.iter())
```

Comment:
Maybe use `itertools::izip!` instead of 3 chained zips? But you would lose the check that they are equal.

Reply:
Yeah I do want to check the length here so will keep this as is